### ISSUE TITLE
Add option to disable cargo space for espionage probes

### ISSUE THREAD

**Issue by lanedirt** [lanedirt/OGameX#206]
Currently the `Espionage Probe` GameObject ship is defined with a cargo capacity of 5.

See:
- `app/GameObjects/CivilShipObjects.php`

However in the official game a restriction has been added where the cargo space of espionage probes is set to 0 when they are used to attack players or transport goods. This is to prevent a game tactic where espionage probes are used to rapidly attack players or transport goods because of their enormous speeds.

**The challenge:**
Currently the codebase only supports "cargo capacity". However, when we would set the "cargo capacity" of espionage probes to 0, it causes issues when dispatching fleets because of deuterium consumption. In a specific part of the fleet dispatch system a second option is mentioned: "fuel capacity". When checking what the official game uses for espionage probes it is seen that it has these values:

- baseFuelCapacity: 5
- baseCargoCapacity: 0

So basically it means we will need to introduce a second config option "fuel capacity" in addition to "cargo capacity".

See `FleetController::dispatchCheckTarget()` which includes a return value for `baseFuelCapacity`. The fleet dispatch screen requires this parameter.
